osfoffline/views/start_screen.py

```diff
@@ -1,12 +1,22 @@
+import furl
+import aiohttp
```

Comment:
I don't necessarily take the default behaviour of PyCharm as best practice, but I'll do it anyway to appease the CR gods.